### PR TITLE
Use standard library in RandomLong/RandomFloat

### PR DESCRIPTION
**Pull request by In-line**
1. `std::random_device` provides seed from entropy, which is better then seeding from `std::time(nullptr)`
2. Code is simplified
3. And of course use of standard library is better.

### CODE REVIEW

**Comment by In-line** [comment]
Actually building was successful, TeamCity failed with weird error.

**Comment by s1lentq** [comment]
@In-line oops, my fault, i'll be fix

File: rehlds/engine/pr_cmds.cpp [hunk 1]
```diff
@@ -28,6 +28,10 @@
 
 #include "precompiled.h"
 
+#ifdef REHLDS_FIXES
+#include <random>
```
Comment (LevShisterov) [inline rehlds/engine/pr_cmds.cpp]: This include should go in osconfig.h.

Comment (In-line) [inline rehlds/engine/pr_cmds.cpp]: @LevShisterov One second

Comment (In-line) [inline rehlds/engine/pr_cmds.cpp]: @LevShisterov Done.

**Comment by WPMGPRoSToTeMa** [comment]
Why close?

**Comment by In-line** [comment]
@WPMGPRoSToTeMa This PR requires newer libstdc++ then in HLDS root. So, we should decide to link libstdc++ statically or bypass this problem another way. For now I close this.

**Pull request closed without merging**